tdrs-backend/tdpservice/search_indexes/reparse.py

```diff
@@ -0,0 +1,110 @@
+"""Reparsing command for selected files."""
+# should include all the steps in the management command
+from tdpservice.data_files.models import DataFile
+from tdpservice.search_indexes.models.reparse_meta import ReparseMeta
+from tdpservice.core.utils import log
+from tdpservice.users.models import User
+from tdpservice.search_indexes.utils import (
+    backup,
+    get_log_context,
+    assert_sequential_execution,
+    should_exit,
+    handle_elastic,
+    delete_associated_models,
+    count_total_num_records,
+    calculate_timeout,
+    handle_datafiles,
```

Comment:
does it make sense to just have all these functions in the `reparse.py` file?

Reply:
I like the idea of that as well